src/Symfony/Component/Asset/Context/NullContext.php

```diff
@@ -18,6 +18,13 @@
  */
 class NullContext implements ContextInterface
 {
+    public function __construct()
+    {
+        if (__CLASS__ !== get_class($this)) {
```

Comment:
@unkind we should discuss about that in that ticket, but I don't see what you mean, declaring a service which class is final and registered as lazy will cause the error, regardless of its interface(s) or parent class(es)